draft-irtf-cfrg-vdaf.md

```diff
@@ -1677,17 +1677,18 @@ are defined in {{prio3-param}}. The methods required for sharding, preparation,
 aggregation, and unsharding are described in the remaining subsections. These
 methods refer to constants enumerated in {{prio3-const}}.
 
-| Parameter         | Value             |
-|:------------------|:------------------|
-| `VERIFY_KEY_SIZE` | `Prg.SEED_SIZE`   |
-| `NONCE_SIZE`      | `16`              |
-| `ROUNDS`          | `1`               |
-| `SHARES`          | in `[2, 256)`     |
-| `Measurement`     | `Flp.Measurement` |
-| `AggParam`        | `None`            |
+| Parameter         | Value                                           |
+|:------------------|:------------------------------------------------|
+| `VERIFY_KEY_SIZE` | `Prg.SEED_SIZE`                                 |
+| `RAND_SIZE`       | `Prg.SEED_SIZE * (1 + 2 * (SHARES - 1))`        |
```

Comment:
One caveat here is that this value of `RAND_SIZE` is larger than we need if no joint randomness is generated. I think this is fine, but depending on we resolve https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/215 we may be able to tighten this.

Reply:
Whoops, I missed that, the size may be too low. Requiring extra randomness might be a good way to clean things up for #215 though.